ember-core/src/main/scala/org/http4s/ember/core/Parser.scala

```diff
@@ -27,7 +27,8 @@ private[ember] object Parser {
         case None =>
           Pull.raiseError[F](
             EmberException.ParseError(
-              s"Incomplete Header received (sz = ${buff.size}): ${buff.decodeUtf8}"))
+              s"""Incomplete Header received (sz = ${buff.size}): utf8: ${buff.decodeUtf8} - b64: "${buff.toBase64}"""")
```

Comment:
This never gets into a response, correct?

The `decodeUtf8` could fail as well, and I don't see a reason to treat it as UTF-8.

Reply:
The buff is a bytevector so if this fails its just a left. But if it is. It makes it much easier to debug.

Reply:
Oh, right, that's an Either.  Yeah, that's fine.

I just wanted to verify that this `ParseError` message doesn't get rendered into a 400, as it can reflect bytes from a request.